partials/expenses_input.html

```diff
@@ -5,8 +5,16 @@
                 <div class="input-title">Expenses</div>
                 <div class="step-subheader">Bills, rent, mortgage interest, and spending every month</div>
                 <div class="input-field-wrapper">
-                    $
-                    <input type="text" autofocus required ng-currency placeholder="3,500" class="steps" ng-model="expenses.value" />
+                    <span class="leading-dollar">$</span>
+                    <input type="text" 
+                    autofocus
+                    required
+                    ng-currency
+                    placeholder="3,500"
+                    class="steps"
+                    min="0"
+                    max="99999"
+                    ng-model="expenses.value" />

```

Comment:
have we figured out why these don't work on iphone yet? I'm guessing it's either the autofocus or the ng-currency directive